src/shared/components/plots/StackedBarPlot.tsx

```diff
@@ -245,8 +259,12 @@ export default class StackedBarPlot extends React.Component<IStackedBarPlotProps
         return { x, y };
     }
 
+    @computed get offset() {
+        return this.barWidth;
+    }
+
     @computed get categoryAxisDomainPadding() {
-        return this.domainPadding;
+        return this.domainPadding + (this.props.stacked ? 0 : this.categoryCoord(this.data.length / 2));
```

Comment:
How did you get to this formula? It seems like a lot of padding.

Reply:
I have done some trial and error, this one was giving perfect plot

Reply:
basically you're making padding take about half the available space, right?

Reply:
yes, adding half the width for each category